runtime/jvmti/jvmtiStackFrame.c

```diff
@@ -311,7 +322,7 @@ jvmtiGetFrameCount(jvmtiEnv* env,
  * At this point we need only ensure that the frame being popped is debuggable.
  * 
  * At this stage we simply set a halt flag in the thread, which will be processed
- * when the thread resumes running.  The remaining logic is handled in 
+ * when the thread resumes running.  The remaining logic is handled in
```

Comment:
There is still a white space change here. You deleted a white space at the end of this line.